source/js/components/petition/petition.jsx

```diff
@@ -1,78 +1,14 @@
 import React from 'react';
 import ReactGA from 'react-ga';
 import classNames from 'classnames';
+import DonationModal from './donation-modal.jsx';
+import FloatingLabelInput from './floating-label-input.jsx';
+import CountrySelect from './country-select.jsx';
 import basketSignup from '../../basket-signup.js';
-import SALESFORCE_COUNTRY_LIST from './salesforce-country-list.js';
-
-class FloatingLabelInput extends React.Component {
```

Comment:
this class and the next kept being incredibly in the way so I put them in their own files.